Zend/tests/class_name_as_scalar_error_005.phpt

```diff
@@ -7,7 +7,7 @@ $x = static::class;
 
 ?>
 --EXPECTF--
-Fatal error: Uncaught Error: Cannot use "static" when no class scope is active in %s:3
+Fatal error: Uncaught Error: Cannot use "static" in the global scope in %s:%d
```

Comment:
`s/in the/from the` maybe?

Reply:
Actually, you seem to also use `from global scope` in some places.

Reply:
Hmm, after having a look at my changes again, I realized why I did it, and I think it was the right choice:
So `in` is used together with `use`, while `from` is used together with `call`. I think `call something from` sounds good, while `use something from` is a bit unnatural.

Another solution could be to write `Cannot call "static" from the global scope`, but I think calling is mainly for functions, so it's probably still not perfect. Are you ok with the current wording or do you still feel that a change is needed?

Reply:
No strong feelings here. From sounds more natural to me personally, but either is okay.